### PR TITLE
Update velero module to compliance with tf > v0.12 SE-2039

### PR DESCRIPTION
**Pull request by rvaralda**

### CODE REVIEW

**Review by floatingatoll — APPROVED** [review]
Seems reasonable enough.

File: aws/velero-bucket/outputs.tf [hunk 1]
```diff
@@ -1,17 +1,17 @@
 output "bucket_name" {
-  value = element(concat(aws_s3_bucket.this.*.id, list("")), 0)
+  value = element(concat(aws_s3_bucket.this.*.id, tolist([""])), 0)
```
Comment (smarnach) [inline aws/velero-bucket/outputs.tf:2]: I'd prefer `try(element(aws_s3_bucket.this.*.id, 0), "")`. I feel that this makes the intent of the expression much clearer. Same for the other outputs.

(Either way, `tolist()` is redundant here.)

Comment (smarnach) [reply aws/velero-bucket/outputs.tf:2]: I think `try(aws_s3_bucket.this.*.id[0], "")` also works.

Comment (rvaralda) [reply aws/velero-bucket/outputs.tf:2]: let me try

Comment (rvaralda) [reply aws/velero-bucket/outputs.tf]: it worked, just updated.

File: aws/velero-bucket/outputs.tf [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 
 output "velero_iam_user_name" {
-  value = element(concat(aws_iam_user.velero_iam_user.*.name, list("")), 0)
+  value = try(aws_iam_user.velero_iam_user.*.name[0], "")
 }
 
 output "velero_iam_user_access_key" {
-  value = element(concat(aws_iam_access_key.velero_iam_access_key.*.id, list("")), 0)
+  value     = try(aws_iam_access_key.velero_iam_access_key.*.id[0], "")
+  sensitive = true
```
Comment (smarnach) [inline aws/velero-bucket/outputs.tf:11]: The key ID isn't a secret, but I  guess it doesn't hurt much either to mark it as one.

**Pull request merged**